pynest/nest/tests/test_connect_arrays_mpi.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+#
+# test_connect_arrays_mpi.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import subprocess as sp
+import unittest
+import nest
+import numpy as np
+
+try:
+    from mpi4py import MPI
+    HAVE_MPI4PY = True
+except ImportError:
+    HAVE_MPI4PY = False
+
+HAVE_MPI = nest.ll_api.sli_func("statusdict/have_mpi ::")
```

Comment:
@jougs this is unrelated to the PR, but I find this is a rather convoluted way of getting this information... shouldn't it be available to the Python level more directly? Is there a good reason for that? (If not I'd raise an issue to discuss it)

Reply:
I agree that this is a bit indirect and am open for further discussion if you opened an issue and mentioned me there.